config/webpack.test.js

```diff
@@ -178,44 +245,78 @@ module.exports = function (options) {
-         */
         {
-          test: /\.woff2?$|\.ttf$|\.eot$|\.svg$/,
-          loaders: [
+          test: /\.component\.less$/,
+          use: [
+            'to-string-loader',
             {
-              loader: "url-loader",
-              query: {
-                limit: 3000,
-                name: 'vendor/fonts/[name].[hash].[ext]'
+              loader: 'css-loader',
+              options: {
+                minimize: false,
+                sourceMap: true,
+                context: '/'
+              }
+            }, {
+              loader: 'less-loader',
+              options: {
+                paths: [
+                  path.resolve(__dirname, "../node_modules/patternfly/dist/less"),
+                  path.resolve(__dirname, "../node_modules/patternfly/dist/less/dependencies"),
+                  path.resolve(__dirname, "../node_modules/patternfly/dist/less/dependencies/bootstrap"),
+                  path.resolve(__dirname, "../node_modules/patternfly/dist/less/dependencies/font-awesome"),
+                ],
+                sourceMap: true
               }
             }
           ]
-        }, {
-          test: /\.jpg$|\.png$|\.gif$|\.jpeg$/,
-          loaders: [
-            {
-              loader: "url-loader",
-              query: {
-                limit: 3000,
-                name: 'vendor/images/[name].[hash].[ext]'
-              }
+        },
+
+        /**
+         * File loader for supporting fonts, for example, in CSS files.
+         */
+        {
+          test: /\.(woff2|woff|ttf|eot|svg)$/,
+          use: {
+            loader: 'url-loader',
+            options: {
+              limit: 3000,
+              includePaths: [
+                path.resolve(__dirname, "../node_modules/patternfly/dist/fonts/")
+              ],
+              name: '_assets/fonts/[name].[ext]'
             }
+          },
+          exclude: [
+            path.resolve(__dirname, "../src/assets/images/"),
+            /OpenSans.*\.(woff2|woff|ttf|eot|svg)$/ // Exclude loading OpenSans fonts as it will come from CDN
           ]
         },
+        {
+          test: /\.(jpg|png|svg|gif|jpeg)$/,
+          use: {
+            loader: 'url-loader',
+            options: {
+              limit: 3000,
+              includePaths: [
+                path.resolve(__dirname, "../src/assets/images/")
+              ],
+              name: '_assets/images/[name].[ext]'
+            }
+          },
+          exclude: path.resolve(__dirname, "../node_modules/patternfly/dist/fonts/")
+        },
 
         /**
          * Raw loader support for *.html
          * Returns file content as string
          *
          * See: https://github.com/webpack/raw-loader
          */
-        {
-          test: /\.html$/,
-          loader: 'raw-loader',
-          exclude: [ path.resolve(__dirname, 'src/index.html') ]
-        },
+        // {
+        //   test: /\.html$/,
```

Comment:
Are these commented lines intentional?

Reply:
commented till we can be sure we don't need it.